wikicharts/content_gap_plots.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #59.        chart.finalize_plot(save_file_name, display=display_flag)
</pre><p>Seems like we could eliminate the whole <code>display</code> parameter and let the default handle it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>